user/deployment/bintray.md

```diff
@@ -114,11 +114,14 @@ The descriptor is in JSON file format in three sections:
 
 #### Package Section
 
-Bintray package information. In case the package already exists on Bintray, only the name, repo and subject fields are mandatory.
+Bintray package information. Following information are mandatories on open source project.
 
 - `name` is the Bintray package name
 - `repo` is the Bintray repository name
 - `subject` is the Bintray subject, which is either a user or an organization
+- `vcs_url` is the Bintray vcs url, could be a github repository url or either vcs repository url
+- `licenses` is the [Bintray licences](https://bintray.com/docs/api/#url_licenses), which is either a list with at list on item.
```

Comment:
Could you elaborate on "which is either a list with at list on item."? Did you mean "which is either a list or an item"? Or "which is a list with at least one item"? Or something else entirely?